tests/integration_tests/document/test_create.py

```diff
@@ -46,8 +46,84 @@ def test_create_document(client: TestClient, data_db: Session, user_header_token
     assert slug.name.startswith("title")
 
 
+def test_create_document_super(
+    client: TestClient, data_db: Session, superuser_header_token
+):
+    setup_db(data_db)
+    new_document = create_document_create_dto(title="Title", family_import_id="A.0.0.3")
+    response = client.post(
+        "/api/v1/documents",
+        json=new_document.model_dump(mode="json"),
+        headers=superuser_header_token,
+    )
+    assert response.status_code == status.HTTP_201_CREATED
+    created_import_id = response.json()
+    actual_fd = (
+        data_db.query(FamilyDocument)
+        .filter(FamilyDocument.import_id == created_import_id)
+        .one()
+    )
+
+    assert actual_fd is not None
+    assert actual_fd.variant_name is not None
+
+    actual_pd = (
+        data_db.query(PhysicalDocument)
+        .filter(PhysicalDocument.id == actual_fd.physical_document_id)
+        .one()
+    )
+    assert actual_pd is not None
+    assert actual_pd.title == "Title"
+
+    slug = (
+        data_db.query(Slug)
+        .filter(Slug.family_document_import_id == actual_fd.import_id)
+        .one()
+    )
+    assert len(slug.name) == len("title") + 1 + 4
```

Comment:
Maybe a comment on the magic numbers?

Reply:
I think the comment should be # 4 = slug hash length 1 = separator

Reply:
Have added SLUG_SUFFIX_LENGTH to represent 4, but not sure what the one refers to..